src/components/map/map.tsx

```diff
@@ -60,16 +63,6 @@ export function Map({
   useMapPin(map, position, layer);
   useMapLegs(map, mapLegs);
 
-  useEffect(() => {
-    if (map.current) {
-      map.current.flyTo({
-        center: position,
-        zoom: initialZoom,
-        speed: 2,
-      });
-    }
-  }, [position, initialZoom]);
-
```

Comment:
This is not needed as the position or initial zoom is not changing unless we show a new route / stop and the map is rendered.